### ISSUE TITLE
adding click

### ISSUE THREAD

**Issue by djarecka** [ReproNim/testkraken#55]
for `runner in testrunner.py

**Comment by kaczmarj** [comment]
is this for a command line interface? i can do this.

**Comment by djarecka** [comment]
yes, I was thinking that would be nice to have an option or running `runner` as a command line, for now I can only use it from python, like [here](https://github.com/ReproNim/testkraken/blob/master/tests/test_basic_examples.py#L14)
(this test should also still work)
